test/functional/apps/visualize/_vega_chart.js

```diff
@@ -18,7 +18,7 @@ export default function ({ getService, getPageObjects }) {
         expect(spyToggleExists).to.be(false);
       });
 
-      it('should have some initial vega spec text', async function () {
+      it.skip('should have some initial vega spec text', async function () {
```

Comment:
Please check your issue #20777 for the issue number on vega, it was already skipped and not part of my PR I just thought I would try it but it failed so I reverted it back to skipped.

Reply:
The Skipped tests issue only looked at master.  That vega test isn't currently skipped on 6.3;
https://github.com/elastic/kibana/blob/6.3/test/functional/apps/visualize/_vega_chart.js#L21

It could be a flaky test and maybe should be skipped on 6.3?

Reply:
Yes you are right on 6.3 it is not skipped, it is flaky since it passed for me initially but then failed on some later run, so I disabled it.

Reply:
OK, if it's not something you can easily fix and need to skip it, please also create an issue with the failure details, screenshot, etc. and reference it here.

Reply:
Let me take a look to see if I can fix.